manifests/v1beta1/installs/katib-cert-manager/certificate.yaml

```diff
@@ -1,4 +1,4 @@
-apiVersion: cert-manager.io/v1alpha2
+apiVersion: cert-manager.io/v1
```

Comment:
Does it require any additional API changes after switching from `v1alpha2` to `v1` ?

cc @yanniszark @kimwnasptd

Reply:
We are using `cert-manager` `v1.3` in the manifests repo.

The `cert-manager` deprecated the following cert-manager APIs from `v1.4`

```
cert-manager.io/v1alpha2
cert-manager.io/v1alpha3
cert-manager.io/v1beta1
acme.cert-manager.io/v1alpha2
acme.cert-manager.io/v1alpha3
acme.cert-manager.io/v1beta1
```

https://cert-manager.io/docs/release-notes/release-notes-1.4/

And the latest `cert-manager` only supports `v1` now.